test/support/conn_case.ex

```diff
@@ -26,6 +26,7 @@ defmodule GscraperWeb.ConnCase do
       import Phoenix.ConnTest
       import GscraperWeb.ConnCase
       import Gscraper.Factory
+      import GscraperWeb.Gettext
```

Comment:
Maybe

```suggestion
      import Gscraper.Factory
      import GscraperWeb.ConnCase
      import GscraperWeb.Gettext
```

Reply:
yes, it must be like that. Fixed in deb8e19